src/docs/asciidoc/MessageBus.asciidoc

```diff
@@ -210,3 +211,66 @@ The following query params are supported:
 * pw - admin password (default `guest`)
 * vhost - the vhost used for the bus resources (default `/`)
 * busPrefix - the prefix used for all bus resources (default `xdbus.`)
+
+[[kafka-bus-partitions]]
+=== Kafka Message Bus Partition Control
+
+This section describes how topic partitioning functions when using Kafka as transport.
+
+[[kafka-partition-control]]
+==== Controlling the partition count of a transport topic
+
+The _KafkaMessageBus_ will attempt to set the number of partitions in a transport topic as _consumerCount * consumerConcurrency_,
+either by creating the topic with the required number of partitions or by repartitioning it in case it exists.
```

Comment:
Actually, 'it' was referring to the topic. Probably a comma would have worked better :)